api/.eslintrc.json

```diff
@@ -25,7 +25,8 @@
       "error",
       {
         "cases": {
-          "camelCase": true
+          "camelCase": true,
```

Comment:
When I moved the webservice files to a folder I was getting a eslint error, I added this rule to allow pascalCase, something we already do.